.golangci.yml

```diff
@@ -83,6 +83,7 @@ linters-settings:
 
   goimports:
     local-prefixes: github.com/0xPolygon/go-ibft
+                    github.com/prometheus/client_golang/prometheus
```

Comment:
Why is this necessary?

Reply:
Not sure why do we have local imports in the first place... But if we want to keep it, then we must expand that list, because of prometheus-client dependency being added.